connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -255,7 +255,23 @@ public ConfigInfos validateConnectorConfig(Map<String, String> connectorProps) {
 
             // do custom connector-specific validation
             Config config = connector.validate(connectorProps);
+            if (null == config) {
+                throw new BadRequestException(
+                    String.format(
+                        "%s.validate() must return a valid Config.",
```

Comment:
What about:

    %s.validate() must return a Config that is not null.

like on [this line](https://github.com/apache/kafka/pull/3762/files#diff-99fd9ec8598ede0037750255a669db4bR233)?